README.md

```diff
@@ -236,20 +236,25 @@ module.exports = {
 }
 ```
 
-Environment variables can also be configured separately for development and production:
+Environment variables can also be configured separately for development, production plus any custom environments you have. The default environment for `sku build` is `production`, however you can select a custom environment to build your application by passing the command line argument `--env` (`-e` for shorthand). The environment is also passed to your code using `process.env.SKU_ENV`. Please note that these environments are not related to `NODE_ENV`.
```

Comment:
Minor nitpick, but I'd reword this as "development and production, plus any custom environments...".